tools/markdown-to-html/docs-marked-renderer.ts

```diff
@@ -90,7 +100,19 @@ export class DocsMarkdownRenderer extends Renderer {
    * Method that will be called after a markdown file has been transformed to HTML. This method
    * can be used to finalize the content (e.g. by adding an additional wrapper HTML element)
    */
-  finalizeOutput(output: string): string {
+  finalizeOutput(output: string, fileName: string): string {
+    // Check if any fragments have been discovered that do not exist in
+    // the rendered file. We want to error for broken fragment links.
+    this._referencedFragments.forEach(id => {
+      if (this._slugger.seen[id] === undefined) {
```

Comment:
Do we want this to be written so that it returns all fragments that do not exist, rather than just the first one?

Reply:
Yep. Good point. Updated!